browser-test/package-lock.json

```diff
@@ -6583,8 +6583,7 @@
       "version": "5.3.2",
       "resolved": "https://registry.npmjs.org/acorn-jsx/-/acorn-jsx-5.3.2.tgz",
       "integrity": "sha512-rq9s+JNhf0IChjtDXxllJ7g41oZk5SlXtp0LHwyA5cejwn7vKmKp4pPri6YEePv2PU65sAsegbXtIinmDFDXgQ==",
-      "dev": true,
-      "requires": {}
```

Comment:
do we need to put this back after the issue is resolved too?  if so add comment.

Reply:
I think it's some odd side-effect of adding legacy-peer-deps=true. This file, package-lock.json is auto-generated during `npm install`. And reading description of `requires` field - it sounds like empty object or missing are the same. So I'm just going to ignore it given it's auto-generate.